sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/CalciteToDBSPCompiler.java

```diff
@@ -2250,7 +2260,7 @@ void visitSort(LogicalSort sort) {
             return;
         }
         if (sort.offset != null)
-            throw new UnimplementedException(node);
+            throw new UnimplementedException("OFFSET in SORT not yet implemented", node);
```

Comment:
The cases where we point to issue 457 are great. It's really clear that way and invites people to leave a comment to ask for support.

Can we add that pattern to some of the other cases (including this one, but also group-by + filter, join types, single aggregate + filter etc.)?

Reply:
Why add issues if no one asked for them? Issues are about prioritizing work.
We have an umbrella issue for unimplemented stuff: https://github.com/feldera/feldera/issues/172

Reply:
Lowers the barrier for someone to come in and leave a comment. Defaulting to #172 looks great actually.